monday.go

```diff
@@ -445,13 +483,24 @@ func Format(dt time.Time, layout string, locale Locale) string {
 	if !ok {
 		return fm
 	}
-	return intFunc(fm, layout)
+	formatted := intFunc(fm, layout)
+	if locale == LocaleArAE {
+		for k, v := range arabicDigits {
+			formatted = strings.ReplaceAll(formatted, k, v)
+		}
+	}
+	return formatted
 }
 
 // ParseInLocation is the standard time.ParseInLocation wrapper, which replaces
 // known month/day translations for a specified locale back to English before
 // calling time.ParseInLocation. So, you can parse localized dates with this wrapper.
 func ParseInLocation(layout, value string, loc *time.Location, locale Locale) (time.Time, error) {
+	if locale == LocaleArAE {
+		for k, v := range arabicDigits {
+			value = strings.ReplaceAll(value, v, k)
+		}
+	}
```

Comment:
Here is my hack for Arabic numbers. If we decide to keep it, later we should probably move this logic to back-end repo, because there are some more things to convert (passengers, stars, etc.)

Reply:
Added 2 reviewers, not sure who knows this stuff better.

Reply:
Removed digits conversion for now, will solve later in other place